### ISSUE TITLE
Default API endpoint to the current instance

### ISSUE THREAD

**Issue by hummelstrand** [jstarpl/nightscout-exporter#1]
I'd love for this to be merged into the official Nightscout master, and I think this functionality would work best if it was implemented as a sub-folder (just like /report ) where the user wouldn't have to fill in any URL.

**Comment by hummelstrand** [comment]
So a "Basic" section where the user only has to pick the date range and the format, without both the API endpoint and the API Secret fields.

**Comment by jstarpl** [comment]
Yeah, I need to look at how the nightscout is built. I am hoping to stick to TypeScript/React since that accelerates development greatly. Rewriting this to jQuery/Vanilla JavaScript would just be difficult to maintain. So maybe this could just be imported as a drop-in component? I'll have to figure out what's the best way to integrate it directly.
